### ISSUE TITLE
Fix accessibilittt problem reported by the Chrome Lighthouse audit

### ISSUE THREAD

**Issue by SuperPat45** [readthedocs/sphinx_rtd_theme#1057]
### Problem

The chrome lighthouse auditor report some accessibility problems;
_ Color contrast problems in the n*lateral nav bar
_ Missing aria-label in the input search box
_ Duplicated access key on the previous and next buttons when variable theme_prev_next_buttons_location == 'both'

### Reproducible Project

The first two are available in all rtd doc like: https://sphinx-rtd-theme.readthedocs.io/en/latest/
the third one need variable theme_prev_next_buttons_location == 'both'

#### Expected Results

Improve Chrome Lighthouse audit score

**Comment by stsewd** [comment]
Hi, thanks for the report, some of these are already reported:

- Contrast https://github.com/readthedocs/sphinx_rtd_theme/issues/971
- Search bar without a label/aria-label https://github.com/readthedocs/sphinx_rtd_theme/issues/970 https://github.com/readthedocs/sphinx_rtd_theme/pull/1024


The third one I don't think it has been reported before.
